mapadroid/madmin/endpoints/api/apks/MadApkEndpoint.py

```diff
@@ -58,7 +61,7 @@ async def post(self):
             elif not file.filename:
                 await self._add_notice_message('No file selected for uploading')
                 raise web.HTTPFound(self._url_for("upload"))
-            elif not allowed_file(file.filename):
```

Comment:
Is `allowed_file` utilized anywhere else? If not, the code should be removed to have a cleaner code base.